src/EditorFeatures/CSharpTest/UsePatternMatching/CSharpIsAndCastCheckTests.cs

```diff
@@ -518,5 +518,25 @@ static void N(ParameterSyntax parameter)
     }
 }");
         }
+
+        [WorkItem(429612, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/429612")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTypeCheck)]
+        public async Task TestMissingWithNullableType()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"
+class C
+{
+    public object Convert(object value)
+    {
+        if (value is bool?)
```

Comment:
:memo: In almost all cases, it's safe to rewrite this as the following:

```csharp
if (value is bool tmp)
{
}
```

The one exception I can think of is a case where the static type of `tmp` is used as a generic argument.